io.github.dfandrich.gpscorrelate.metainfo.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should be sent upstream if possible

Reply:
It's in process, but I wanted to build the current version right now.